servicebox-app/src/main/java/io/halkyon/services/ClaimStatus.java

```diff
@@ -1,8 +1,14 @@
-package io.halkyon.service;
+package io.halkyon.services;
 
 public enum ClaimStatus {
+    // When claim is created
     NEW,
+    // When claim is waiting for a service to be available
     PENDING,
+    // When claim has a linked service
+    CLAIMED,
+    // When the claim exceeded the attempts to find an available service.
+    ERROR,
     REJECTED,
     BIND;
```

Comment:
I asked about the rejected and bind statuses in here: https://github.com/halkyonio/primaza-poc/discussions/59

Reply:
ok, looking at it